ios/RNIapIosSk2.swift

```diff
@@ -711,7 +711,20 @@ class RNIapIosSk2iOS15: Sk2Delegate {
                     }
                     debugMessage("Purchase Started")
 
-                    let result = try await product.purchase(options: options)
+                    guard let windowScene = await currentWindow()?.windowScene else {
+                        reject(IapErrors.E_USER_CANCELLED.rawValue, "Could not find window scene", nil)
```

Comment:
This should not count as a user cancellation error though

Reply:
Good catch 👍🏻